test/uk/gov/hmrc/emcstfe/fixtures/GetMovementIfChangedFixture.scala

```diff
@@ -611,6 +614,15 @@ trait GetMovementIfChangedFixture extends BaseFixtures with TraderModelFixtures
     )
   )
 
-  val getMovementIfChangedMongoResponse = GetMovementMongoResponse(testArc, JsString(getMovementIfChangedResponseBody))
+  val getMovementIfChangedMongoResponse: GetMovementMongoResponse = GetMovementMongoResponse(testArc, JsString(getMovementIfChangedResponseBody))
+
+  lazy val getRawMovementIfChangedJson: JsValue = Json.obj(
```

Comment:
Dont think movement if changed wont exist in this format, it will only be get movement